src/components/Categories.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { status } from '../redux/categories/categories';
 
-const Categories = () => (
-  <div>Nothing to see here</div>
-);
+const Categories = () => {
```

Comment:
- [ ] OPTIONAL:  Good job adding the "check status" button and dispatching its corresponding action! I would suggest you actually render the status when pressing that button like this:

```JAVASCRIPT
const Categories = () => {
  const dispatch = useDispatch();
  const stats = useSelector((state) => state.categoriesReducer);
  return (
    <div>
      <button type="button" className="submitBtn" onClick={() => dispatch(status())}>
        Check Status
      </button>
      {stats}
    </div>
  );
};
```

That way, the status will be rendered! 💪

Reply:
Thank you very much @Meltrust I will implement this